src/RemoteSsh.cpp

```diff
@@ -110,7 +110,7 @@ void SshConnection::connect()
         }
 
         mSocketDir = new SpacelessQTemporaryDir();
-        mMasterSocket = mSocketDir->path() + "/ssh_socket";
+        mLeaderSocket = mSocketDir->path() + "/ssh_socket";
```

Comment:
I think `Main` fits better than `Leader`.
Or how about using `control`? `mControlSocket`

Reply:
In this case control seems better.. let me update